File: py/FizzBuzz.py

for i in range(1, 101):

    x = i

    if i%3 == 0 and i%5 == 0:
        x = "FizzBuzz"
    elif i%3 == 0:
        x = "Fizz"
    elif i%5 == 0:
        x = "Buzz"
    
    print(x)

File: swift/FizzBuzz.swift
#!/bin/swift

for i in 1...100 {
    if i%3 == 0 && i%5 == 0 {
        print("\(i) - FizzBuzz")
    } else if i%3 == 0 {
        print("\(i) - Fizz")
    } else if i%5 == 0 {
        print("\(i) - Buzz")
    } else {
        print("\(i) - \(i)")
    }
}
